skabase/SKAAlarmHandler/SKAAlarmHandler.py

```diff
@@ -125,7 +125,7 @@ def read_statsNrAlerts(self):
         # PROTECTED REGION ID(SKAAlarmHandler.statsNrAlerts_read) ENABLED START #
         """
         Reads number of active alerts.
-        :return: Number of active alerts
+        :return:state Number of active alerts
```

Comment:
Why this new word is added here?

Reply:
Removed it. The commit id is: [adc714ab09519e1c760451818fdfc20650de1c7e](https://github.com/ska-telescope/lmc-base-classes/commit/adc714ab09519e1c760451818fdfc20650de1c7e)